Update PROJECT_NAME_sandbox_stage.md (#274)
ossf/tac: process/templates/PROJECT_NAME_sandbox_stage.md

The project must have a minimum of two maintainers with different organizational affiliations.
  * "name, affiliation, GitHub ID"

### Existing TI alignment

> The project should be aligned with an existing OpenSSF Working Group, SIG, or Project (TI).  The project commits to providing the hosting TI quarterly updates on progress.

Reviewer: ```suggestion
Most projects will be sponsored by an existing OpenSSF Working Group, although in some cases a project may be sponsored by the TAC. The project commits to providing the sponsor quarterly updates on progress.
```

Reviewer: I think the alignment part is covered by "Mission of the project" below, and it makes sense to have a section explicitly about the sponsor.

Note that this is different than what we list on https://github.com/ossf/tac/blob/main/process/project-lifecycle.md where we say "Projects must seek one TAC sponsor and one WG sponsor" - but I'm pretty sure we actually want that to say "Projects must seek one TAC sponsor or one WG sponsor" to align with https://github.com/ossf/tac/blob/main/process/sig-lifecycle.md.

... speaking of which on the SIG Lifecycle docs we call this a "governing body" not a "sponsor" like we do on the Project Lifecycle docs. We should probably also standardize this naming, but I think it makes sense to keep this Project Template consistent with the wording on the Project Lifecycle doc.

Reviewer: Oh! I should add that I don't think TI is appropriate here because it TIs can include SIGs or other Projects, and I think we only want Projects reporting to the TAC or a WG, not other SIGs (or other Projects!)

>   * "name of TI that will host this project"

Reviewer: ```suggestion
  * "name of the project sponsor: either a Working Group or the TAC"
```

> Most projects will be sponsored by an existing OpenSSF Working Group, although in some cases a project may be sponsored by the TAC. The project commits to providing the sponsor quarterly updates on progress.
>   * "name of the project sponsor: either a Working Group or the TAC"

Reviewer 2: The requirement that they provide regular updates literally means that we are asking them to **report** to the WG or TAC! :-)
So, we shouldn't shy away from what this relationship means and I think we should just go with:
```suggestion
Most projects will report to an existing OpenSSF Working Group, although in some cases a project may report directly to the TAC. The project commits to providing quarterly updates on progress to the group they report to.
  * "name of the group the project reports to: either a Working Group or the TAC"
```

> 

Reviewer 2: I think this is using the term "sponsor" in a way that is different from what we've used it so far. Indeed, we've used the term to refer to a *person*. For instance, the project lifecycle states that a project "Receives a TAC and WG sponsor for guidance on technical direction."
This text uses "sponsor" to refer to the entity the project reports to.

Reviewer: This is exactly the question - I think we're trying to describe the reporting relationship. For SIGs we use the term "governing body" (which feels maybe too formal, but at least is clear). For projects I though we were using the term "sponsor" in the same way we use "governing body" for SIGs.

Maybe @SecurityCRob can clarify? Are we referring to the reporting relationship here? Or is a sponsor something else that is separate from the reporting relationship?

Reviewer 2: I never used it that way but maybe it's a matter of interpretation and we all have different perspectives on that one...
I too feel that "governing body" is a bit strong, that's why I phrased the above as "the entity the project reports to" instead. :-) It seems a bit softer to me.

Reviewer 3: How about "deliverable" as a better word for this - as in "projects are deliverables of working groups..."

Author: > This is exactly the question - I think we're trying to describe the reporting relationship. For SIGs we use the term "governing body" (which feels maybe too formal, but at least is clear). For projects I though we were using the term "sponsor" in the same way we use "governing body" for SIGs.
> 
> Maybe @SecurityCRob can clarify? Are we referring to the reporting relationship here? Or is a sponsor something else that is separate from the reporting relationship?

Reporting relationship

### Project References
The project should provide a list of existing resources with links to the repository, and if available, website, a roadmap, contributing guide, demos and walkthroughs, and any other material to showcase the existing breadth, maturity, and direction of the project.

| Reference           | URL |
|---------------------|-----|
| Repo                |     |

> | Meeting Agenda      |     |
> | OSSF Calendar Entry |     |

Reviewer: ```suggestion
```

These tables are hard! I can see why it'd be useful to have meeting agendas, if available, but after spending several hours reading these doc I *just now* realized that anything beyond the repository is optional. We don't reference an agenda / calendar meeting though, so I think it makes sense to remove it since it isn't clear if it's optional.

Reviewer 2: I agree. My initial intention with this stuff was to keep it as short as possible. My feeling was that a link to the main TI's page/repo should provide most of what we need. If it doesn't then we can then ask for additional info when we do the review.
